### DISCUSSION TITLE
Add support for #utm_ in addition to ?utm_

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1985] [Anything Else]
"If you’re setting campaign parameters on URLs, please, please use `#utm_` instead of `?utm_`. It saves a lot of crawls on your site, for search engines to find out it doesn’t actually change anything."

From https://twitter.com/raymonmens/status/1543343713753210880

**Comment by rmens** [comment]
My tweets auto-delete, so here is the source. He is quite authoritative. Joost de Valk, creator of Yoast SEO.

https://twitter.com/jdevalk/status/1543308772784881668
